pkg/network/common/egress_dns.go

```diff
@@ -116,39 +143,41 @@ func (e *EgressDNS) Sync() {
 	}
 }
 
-func (e *EgressDNS) GetMinQueryTime() (time.Time, ktypes.UID, string, bool) {
+func (e *EgressDNS) GetMinQueryTime() (time.Time, string, []PolicyUpdate, bool) {
 	e.lock.Lock()
 	defer e.lock.Unlock()
+	policyUpdates := make([]PolicyUpdate, 0)
+	tm, dnsName, timeSet := e.dns.GetMinQueryTime()
+	if !timeSet {
+		klog.Warningf("Time was not set for dns name: %s", dnsName)
```

Comment:
If `timeSet` is `false` then `dnsName` will be `""`. But this is not an error anyway, it just means there are no DNS names in use and so we don't need to poll again until there are.

Reply:
Agreed, will change the log message to be more appropriate.

Reply:
(There doesn't need to be a log message at all.)